src/shared/components/CountdownTimer/api/constants/periods.ts

```diff
@@ -0,0 +1,13 @@
+enum Periods {
+  DAYS = 'days',
+  HOURS = 'hours',
+  MINUTES = 'minutes',
+  SECONDS = 'seconds',
+}
+
+export const periods = [
+  Periods.DAYS,
+  Periods.HOURS,
+  Periods.MINUTES,
+  Periods.SECONDS,
+];
```

Comment:
you don’t need to keep this enum into api folder. Just in CountdownTimer

Reply:
done